### PR TITLE
feat(editor): Alert design system component

### PR DESCRIPTION
**Pull request by cstuncsik**

### CODE REVIEW

File: packages/design-system/src/components/N8nAlert/Alert.vue [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+			[$style.alert]: true,
+			[$style.success]: type === 'success',
+			[$style.error]: type === 'error',
+			[$style.warning]: type === 'warning',
+			[$style.info]: type === 'info',
+			[$style.isLight]: effect === 'light',
+			[$style.isDark]: effect === 'dark',
+			[$style.center]: center,
+			[$style.background]: background,
+		}"
```
Comment (alexgrozav) [inline packages/design-system/src/components/N8nAlert/Alert.vue:14]: nit: Could be extracted into computed `classes` variable.

**Review by alexgrozav — DISMISSED** [review]
Looking great! One small remark. Nice work, @cstuncsik!

File: packages/design-system/src/components/N8nAlert/Alert.vue [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+		role="alert"
+	>
+		<div :class="$style.content">
+			<span v-if="showIcon" :class="$style.icon">
+				<n8n-icon :icon="icon" />
```
[the remark this answers is not in the document]
Comment (cstuncsik) [reply packages/design-system/src/components/N8nAlert/Alert.vue]: done

**Comment by cstuncsik** [comment]
Thanks for the nice tips @alexgrozav , applied them all!

**Review by alexgrozav — APPROVED** [review]

**Comment by janober** [comment]
Got released with `n8n@0.206.1`

**Comment by janober** [comment]
Got released with `n8n@0.206.1`

**Pull request merged**